work_queue/src/work_queue_worker.c

```diff
@@ -869,17 +872,18 @@ static int do_task( struct link *master, int taskid, time_t stoptime )
 			}
 			free(env);
 		} else if(!strcmp(line,"end")) {
-			work_queue_process_compute_disk_needed(p);
+			//work_queue_process_compute_disk_needed(p);
```

Comment:
Ok, that's fine, but don't leave the dead code in place.
Presumably work_queue_process_compute_disk_needed ought to be called at some later point?